master/internal/configpolicy/task_config_policy.go

```diff
@@ -2,32 +2,18 @@ package configpolicy
 
 import (
 	"github.com/determined-ai/determined/master/pkg/model"
+	cp "github.com/determined-ai/determined/master/pkg/schemas/configpolicy"
 	"github.com/determined-ai/determined/master/pkg/schemas/expconf"
 )
 
-// ResourceConstraints are non-overridable resource constraints.
-// Submitted workloads that request resource quanities exceeding defined resource constraints in a
-// given scope are rejected.
-type ResourceConstraints struct {
-	MaxSlots *int `json:"max_slots"`
-}
-
-// Constraints are non-overridable workload constraints.
-// Submitted workloads whose config's respective field(s) exceed defined constraints within a given
-// scope are rejected.
-type Constraints struct {
-	ResourceConstraints *ResourceConstraints `json:"resources"`
-	PriorityLimit       *int                 `json:"priority_limit"`
-}
-
```

Comment:
This is being moved to `master/pkg/schemas/configpolicy/task_config_policy.go` right? When do we put something in `internal` vs `pkg`?

Was there an import conflict with having the constraint definition in `internal`? Is there a reason not to move this entire file to `pkg`?

Reply:
Yea! There were some import cycles occurring if we moved`ExperimentConfigPolicy` and `NTSCConfigPolicy` into here because those use `model.CommandConfig`, and model uses `configpolicy.Constraints`.
It's probably a lot cleaner if I move them over into `master/internal/pkg/model/task_config_policy.go` with the other Go structs used to query the db! 
Thanks for pointing this out!